packages/owl-bot/src/owl-bot-yaml-schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",

```

Comment:
we should confirm that `compile` moves this JSON into the `build/` folder, otherwise this might cause issues on deployment.